modules/ROOT/pages/authentication-authorization/dbms-administration.adoc

```diff
@@ -1070,6 +1070,10 @@ GRANT [IMMUTABLE] DATABASE MANAGEMENT
 
 |===
 
+From Cypher 25, the `ALTER DATABASE`, `SET DATABASE ACCESS`, `SET DATABASE DEFAULT LANGUAGE`, and `ALTER COMPOSITE DATABASE` privileges can be granted both as database privileges and as DBMS privileges.
```

Comment:
1. is this a good enough marker for when it was introduced? 
2. does it also need neo4j version?

I assume whatever the answer is here that it should also apply to the other ones in this file

Reply:
2. probably? but how do we nicely fit that in? (or do we only say 2025.08 instead of Cypher 25?)